src/sonic-yang-models/yang-models/sonic-loopback-interface.yang

```diff
@@ -63,12 +63,11 @@ module sonic-loopback-interface {
                 key "name ip-prefix";
 
                 leaf name{
-                    /* This node must be present in LOOPBACK_INTERFACE_LIST */
-                    must "(current() = ../../LOOPBACK_INTERFACE_LIST[name=current()]/name)"
-                    {
-                        error-message "Must condition not satisfied, Try adding lo<>: {}, Example: 'lo1': {}";
+                    description "Loopback interface name";
+
+                    type leafref {
+                        path "../../LOOPBACK_INTERFACE_LIST/name";
```

Comment:
Lets keep error-msg, something as below: 

error-message "Loopback VRF interface must exist, Try adding lo<>: {}, Example: 'lo1': {}";
description "Loopback interface name";

Reply:
plz see if we must do similar changes in VLAN_INTERFACE and in INTERFACE. Thx

Reply:
I haven't seen `error-message` used with leafref before in any sonic yang models. I think it is not supported:
```
libyang[0]: Invalid keyword "error-message". (path: /sonic-loopback-interface:sonic-loopback-interface/LOOPBACK_INTERFACE/LOOPBACK_INTERFACE_IPPREFIX_LIST/name)
```

Tried like this:
```
                leaf name{
                    error-message "Loopback VRF interface must exist, Try adding lo<>: {}, Example: 'lo1': {}";

                    description "Loopback interface name";

                    type leafref {
                        path "../../LOOPBACK_INTERFACE_LIST/name";
                    }
                }
```

and this:
```
                leaf name{
                    description "Loopback interface name";

                    type leafref {
                        error-message "Loopback VRF interface must exist, Try adding lo<>: {}, Example: 'lo1': {}";
                        path "../../LOOPBACK_INTERFACE_LIST/name";
                    }
                }
```

Reply:
> plz see if we must do similar changes in VLAN_INTERFACE and in INTERFACE. Thx

Yes, we might need to fix these as well. I will start first by checking if we can can the referrers from must statements #9534